auth/src/test/java/com/redhat/lightblue/rest/auth/ldap/LdapRoleProviderIntegrationTest.java

```diff
@@ -102,4 +103,11 @@ public void testUserWhoDoesNotExist() throws Exception {
         assertTrue(provider.getUserRoles("idontexist").isEmpty());
     }
 
+    @Test
+    public void testHealthCheck() throws Exception {
+        
+        RolesProviderHealth health = provider.checkHealth();
+        System.out.println(health.details());
+        assertTrue(health.isHealthy());
+    }
```

Comment:
You can maybe do what you did in the Mongo one and try to connect on a different port than the one being used by in-memory LDAP server